app/design/frontend/Magento/blank/web/css/source/components/_modals_extend.less

```diff
@@ -152,6 +152,7 @@
 
 .media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__m) {
     .modal-popup {
+        pointer-events: auto;
```

Comment:
@priti2jcommerce Could you add extra empty line after that in both files to follow coding standards? :) 

Thanks,